be/src/runtime/descriptors.h

```diff
@@ -210,9 +210,14 @@ class IcebergTableDescriptor : public HiveTableDescriptor {
     ~IcebergTableDescriptor() override = default;
     bool has_partition() const override { return false; }
     const TIcebergSchema* get_iceberg_schema() const { return &_t_iceberg_schema; }
+    bool is_unpartitioned_table() { return _partition_column_names.empty(); }
```

Comment:
maybe override `has_paritition()` in base class

Reply:
if I override `has_partition()`, there will be a lot of adaptation work during iceberg scan stage.